libs/category/testing/src/inmemory-backend/category.service.ts

```diff
@@ -63,30 +64,21 @@ export class DaffInMemoryBackendCategoryService implements InMemoryDbService {
     });
 	}
 	
-	private getTotalPages(allIds: number[], pageSize: number) {
-		return Math.floor(allIds.length / pageSize) + 1;
+	private getTotalPages(allIds: string[], pageSize: number) {
+		return allIds.length % pageSize ? Math.floor(allIds.length / pageSize) + 1: allIds.length/pageSize;
 	}
 
-	private trimProductIdsToSinglePage(allIds: number[], currentPage: number, pageSize: number) {
+	private trimProductIdsToSinglePage(allIds: string[], currentPage: number, pageSize: number) {
```

Comment:
We don't care about the type. This can either be any, or you can make this generic.

Reply:
done